filters/auth/grantflowstate.go

```diff
@@ -0,0 +1,132 @@
+package auth
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/zalando/skipper/secrets"
+)
+
+const randomStringLength = 20
```

Comment:
ignore: the const name is funny, because the string length is quite static :)

Reply:
Removed. Unused after removing the `randString()` function